packages/wallet-connection/agoric-wallet-connection.js

```diff
@@ -1,2 +1,4 @@
-export * from '@agoric/web-components/agoric-wallet-connection.js';
-console.warn('This package is deprecated. Use @agoric/web-components instead of @agoric/wallet-connection.');
+export * from "@agoric/web-components/agoric-wallet-connection.js";
+console.warn(
+  "This package is deprecated. Use @agoric/web-components instead of @agoric/wallet-connection."
```

Comment:
double quotes? What's going on here?

Reply:
When I added the `pretty-fix` to the packages that were missing it, I neglected to add the `prettier` config block to each of them too.

In 270796b I removed them all to define in one place, at the root.